src/pages/case-studies/ibm-commerce-platform.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: IBM Commerce Platform
+description:
+  This case study examines the adoption of Carbon for the design and development
+  of the new self-service purchase experience on IBM.com.
+---
+
+<PageDescription>
+
+Learn about how the IBM.com team adopted Carbon for the design and development
+of the new self-service purchase experience on IBM.com.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>The challenge</AnchorLink>
+<AnchorLink>The proposed solution</AnchorLink>
+<AnchorLink>The approach</AnchorLink>
+<AnchorLink>Benefits</AnchorLink>
+<AnchorLink>Positive outcomes</AnchorLink>
+<AnchorLink>Next steps</AnchorLink>
+<AnchorLink>Resources</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+The IBM.com Commerce Platform team spent three months transitioning their
+checkout flow to implement Carbon and improve their conversion rate and overall
+product experience. They took an iterative approach, starting first with small,
+isolated features to get comfortable designing and developing with Carbon. Next,
+they replaced existing elements with Carbon components. Once all the components
+had been transitioned, the team focused on updating page structures and layouts.
+
+Since the Carbon implementation, the team’s delivery cycle has sped up and
+they spend less time on QA. Their product is now aligned with the
```

Comment:
Any need to spell out quality assurance and not use the abbreviation QA? Same goes for UI below? Or do we think our audience will know what's up? I mean, we don't really say "quality assurance" or "user interface" in our day to day... so the abbreviation are probably fine. Just checking!

Reply:
Good catch, Matt! We're going to write out the abbreviations. 🙂